### DISCUSSION TITLE
How to Identify Oscillating Trends

### DISCUSSION THREAD

**Discussion by tiger2014** [DaveSkender/Stock.Indicators#874] [Help and support]
This is not a technical question, but I have asked it in many places without an answer.  So, I hope someone here can help to tell me how to identify an oscillator trend with indicators. It has bothered me for a long time. I had used CCI and ADX, but they didn't work.

The Oscillator Trend like this:

![image](https://user-images.githubusercontent.com/6703612/185774383-ca0f3481-6b51-46f5-aeb5-942379a7805e.png)

Thanks a lot!

**Comment by DaveSkender** [comment]
What's your definition of an "oscillator trend"?  Is it relatively flat but moving back and forth with a certain sine wave pattern?

**Reply by tiger2014** [reply]
I always use a 5m timeframe to trade ES. So my meaning of  "oscillator trend" is 'relatively flat but moving back and forth with a certain sine wave pattern' in a 5m timeframe. I want to identify them and use another strategy that is different from a trend strategy.

**Reply by DaveSkender** [reply]
You might try [slope](https://daveskender.github.io/Stock.Indicators/indicators/Slope/#content) over ~30 bars, where `Slope` is between +/- 1.  It returns `StdDev` too, so you could measure for a minimum volatility number as well.
